### PR TITLE
Test and update screenshot - Issue #2199

### PR DESCRIPTION
**Pull request by ruthryi**
Reviewed and Tested : https://apim.docs.wso2.com/en/latest/develop/customizations/customizing-the-developer-portal/styling-the-logo-and-header/

Tested
- https://apim.docs.wso2.com/en/latest/develop/customizations/customizing-the-developer-portal/overriding-developer-portal-theme/
- https://apim.docs.wso2.com/en/latest/administer/multitenancy/managing-tenants/#managing-tenants

Fixes https://github.com/wso2/docs-apim/issues/2199

### CODE REVIEW

File: en/docs/develop/customizations/customizing-the-developer-portal/styling-the-logo-and-header.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ const Configurations = {
 #### Example
 
 We can change the logo and header background as follows by changing the above parameters.
```
Comment (shaniR) [inline en/docs/develop/customizations/customizing-the-developer-portal/styling-the-logo-and-header.md:27]: above parameters or below parameters?

Comment (shaniR) [reply en/docs/develop/customizations/customizing-the-developer-portal/styling-the-logo-and-header.md:27]: Why do we have a configuration on top?

Comment (shaniR) [reply en/docs/develop/customizations/customizing-the-developer-portal/styling-the-logo-and-header.md:27]: What is the file we need to change? Where can we find it? Shall we add these steps in the doc too?

**Review by shaniR — CHANGES_REQUESTED** [review]

File: en/docs/develop/customizations/customizing-the-developer-portal/styling-the-logo-and-header.md (hunk 1 above)
Comment (ruthryi) [reply en/docs/develop/customizations/customizing-the-developer-portal/styling-the-logo-and-header.md]: Updated in the following commits

**Review by shaniR — APPROVED** [review]

**Pull request merged**